### DISCUSSION TITLE
Thanks for producing jupyterbook

### DISCUSSION THREAD

**Discussion by FriendlyUser** [executablebooks/meta#104] [General] [answered]
I was wondering when jupyterbook 1.0.0 is expected to be released, I would love to start using it for a technical book I want to start working on. This will use a lot of python code and examples.

And I cannot wait to start using myst to simplify writing rst, I could not remember rst syntax.

**Comment by chrisjsewell** [comment, marked as answer]
Heya,
I would say most of the core implementation and user interface are now pretty solid, so we will certainly not be looking to make many radically breaking changes now (or at least will clearly highlight them); just more improvements, bug fixes, additional features etc.
I wouldn't be too worried about starting to use v0.8 for your book then.

As to an actual 1.0.0 release, well we don't want to rush it through, so I would not expect it too soon. Maybe early next year, but that's pure speculation.

**Reply by FriendlyUser** [reply]
Just curious about when it will be possible to customize the latex template used to create a pdf, I guess I could manually modify it after spitting it out with `jb build book latex`

**Reply by chrisjsewell** [reply]
@mmcky ans @AakashGfude are doing lots of work on the LaTeX/PDF front, so hopefully not too long 🤞
